basic/src/test/java/org/jboss/hal/testsuite/test/configuration/messaging/destinations/ConnectionFactoriesTestCase.java

```diff
@@ -49,60 +45,46 @@
     private MessagingPage page;
 
     @BeforeClass
-    public static void setUp() {
-        dispatcher = new Dispatcher();
-        verifier = new ResourceVerifier(dispatcher);
+    public static void setUp() throws Exception {
+        createConnectionFactory(CONN_FACTORY_TBR_ADDRESS, "java:/" + CONNECTION_FACTORY_TBR, CONNECTOR_TBA);
+        administration.reloadIfRequired();
     }
 
     @Before
     public void before() {
-        if (ConfigUtils.isDomain()) {
-            address = new ResourceAddress(domainPath);
-            command = DOMAIN + ADD;
-            remove = DOMAIN + remove;
-        } else {
-            address = new ResourceAddress(path);
-            command = ADD;
-        }
+        page.navigateToMessaging();
+        page.selectQueuesAndTopics();
+        page.switchToConnectionFactories();
     }
 
     @After
-    public void after() {
-        cliClient.executeCommand(remove);
+    public void after() throws InterruptedException, TimeoutException, IOException {
+        administration.reloadIfRequired();
     }
 
     @AfterClass
-    public static void tearDown() {
-        dispatcher.close();
+    public static void tearDown() throws IOException, OperationException {
+        operations.removeIfExists(CONN_FACTORY_TBA_ADDRESS);
+        operations.removeIfExists(CONN_FACTORY_TBR_ADDRESS);
     }
 
-    @Test //https://issues.jboss.org/browse/HAL-832
-    public void addConnectionFactory() {
-        page.navigateToMessaging();
-        page.selectQueuesAndTopics();
-        page.switchToConnectionFactories();
-
-        page.addFactory(NAME, JNDINAME, CONNECTOR);
-
-        verifier.verifyResource(address, true);
-
-        cliClient.executeCommand(remove);
-
-        verifier.verifyResource(address, false);
+    @Test
+    public void addConnectionFactory() throws Exception {
+        page.addFactory(CONNECTION_FACTORY_TBA, JNDI_NAME_TBA, CONNECTOR_TBA);
+        new ResourceVerifier(CONN_FACTORY_TBA_ADDRESS, client).verifyExists();
     }
 
     @Test
-    public void removeConnectionFactory() {
-        cliClient.executeCommand(command);
-
-        page.navigateToMessaging();
-        page.selectQueuesAndTopics();
-        page.switchToConnectionFactories();
-
-        verifier.verifyResource(address, true);
-        page.selectInTable(NAME, 0);
-        page.remove();
+    public void removeConnectionFactory() throws Exception {
+        page.remove(CONNECTION_FACTORY_TBR);
+        new ResourceVerifier(CONN_FACTORY_TBA_ADDRESS, client).verifyDoesNotExist();
+    }
 
-        verifier.verifyResource(address, false);
+    private static void createConnectionFactory(Address address, String jndiName, String connector) throws Exception {
+        operations.add(address, Values.empty()
```

Comment:
Is there any particular reason why not use `ofList` method instead of `empty` one? Not really important just may be a bit confusing.